utils.py

```diff
@@ -126,6 +126,20 @@ def get_config(config_key):
             'description': 'The whole test analysis with all results',
             'type': 'list of items'
         },
+        {
+            'id': 7,
+            'name': 'Weighted scores',
+            'short_name': 'weighted_s',
```

Comment:
better to keep full name like `weighted_scores`

Reply:
Oh yes. I will update.